src/OXRS_WT32.cpp

```diff
@@ -167,16 +171,17 @@ void _getConfigSchemaJson(JsonVariant json)
     _mergeJson(properties, _fwConfigSchema.as<JsonVariant>());
   }
 
-  // SHT20 sensor config
-  if (_climateSensorFound)
+  // sensor config
+#if defined(CONFIG_IDF_TARGET_ESP32S3)
```

Comment:
Won't this mean we only see the config for ESP32S3s and not screens with an SHT20 connected?

Reply:
There is only a single config for all sensors now.
The config is enabled for all S3 chips because only they have the builtin sensor.
All supported panels are based on S3 chips, except the classic WT32. 
The SHT is only included in the payload if found.

I simplified a little so an SHT20 connected to the classic WT32-SC01 is not supported.

Reply:
But if an SHT20 is connected to the classic WT32-SC01, you will still end up publishing telemetry data, there just won't be any config shown.